content/github/site-policy/github-insights-and-data-protection-for-your-organization.md

```diff
@@ -39,8 +39,8 @@ Under various data protection regulations, such as the General Data Protection R
 
 - **Exclusion of processing**: Users may have the right to have their personal data excluded from being processed. Administrators have the ability to remove a contributor’s data from being processed in {% data variables.product.prodname_insights %}, and the resulting reports and metrics will exclude the contributor’s data accordingly. For more information, see "[Managing contributors and teams](/insights/installing-and-configuring-github-insights/managing-contributors-and-teams)."
 
-- **Access**: Users may have the right to demand to see what personal data is being processed. Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)." Raw data is available through the {% data variables.product.prodname_enterprise %} API. Your organization is responsible for any decisions to process personal data and for fulfilling any such requests.
+- **Access**: Users may have the right to demand to see what personal data is being processed. Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](https://docs.github.com/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)." Raw data is available through the {% data variables.product.prodname_enterprise %} API. Your organization is responsible for any decisions to process personal data and for fulfilling any such requests.
 
 - **Correction and deletion**: Users may have the right to rectify or delete their personal data. The data used in {% data variables.product.prodname_insights %} is derived from the existing data you add to or generate from your {% data variables.product.prodname_ghe_server %} installation. Correction and deletion should follow your organization's existing process to correct and delete data from {% data variables.product.prodname_ghe_server %}.
 
-- **Transparency regarding processing**:  Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)."
+- **Transparency regarding processing**:  Each metric and report has a detailed description of what personal data is being processed. For more information, see "[Metrics available with {% data variables.product.prodname_insights %}](https://docs.github.com/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights)."
```

Comment:
Help

Reply:
Yeah, this /insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights leads to  https://docs.github.com/en/free-pro-team@latest/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights

but https://docs.github.com/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights leads to the actual page which is https://docs.github.com/en/enterprise-server@2.22/insights/exploring-your-usage-of-github-enterprise/metrics-available-with-github-insights.